hi-ml-azure/testazure/testazure/test_himl.py

```diff
@@ -875,20 +876,13 @@ def spawn() -> Tuple[int, List[str]]:
         if run.status not in ["Failed", "Completed", "Cancelled"]:
             run.wait_for_completion()
         assert run.status == "Completed"
-        log_root = path / "logs"
-        log_root.mkdir(exist_ok=False)
-        files = run.get_file_names()
-        # Account for old and new job runtime: log files live in different places
-        driver_log_files = ["azureml-logs/70_driver_log.txt", "user_logs/std_log.txt"]
-        driver_log = log_root / "driver_log.txt"
-        for f in driver_log_files:
-            if f in files:
-                run.download_file(f, output_file_path=str(driver_log))
-                break
-        else:
-            raise ValueError("The run does not contain any of the driver log files")
-        log_text = driver_log.read_text()
-        return log_text
+
+        # TODO: upgrade to walrus operator when upgrading python version to 3.8+
+        # if log_text := get_driver_log_file_text(run=run):
+        log_text = get_driver_log_file_text(run=run)
+        if log_text:
```

Comment:
Not sure about this, but is it maybe better to explicitly say `if log_text is None`?

Reply:
I think it is. Imagine the log exists, but it's blank. This would still unexpectedly fail.